TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java

```diff
@@ -55,7 +55,7 @@ public class CaptureProxy {
     private static final String HTTPS_CONFIG_PREFIX = "plugins.security.ssl.http.";
     public static final String DEFAULT_KAFKA_CLIENT_ID = "HttpCaptureProxyProducer";
 
-    static class Parameters {
+    public static class Parameters {
         @Parameter(required = false,
```

Comment:
@gregschohn @AndreKurait I think the parameters themselves need to be `public` as well otherwise we can't access the parameters in our subclass of `CaptureProxy`. Please see the commit from my fork which contains all the necessary fields that need to be marked `public`/`protected`: https://github.com/opensearch-project/opensearch-migrations/commit/fb082659cfc197061e6d8e17428edf4e96567090

Reply:
Updated, thanks!